### PR TITLE
docs(topology): update README.md

### LINKED ISSUE #677: Update topology plugin documentation
### Description
The current documentation for enabling code decorators to navigate to Git repositories (or open a Openshift Dev Space) states to add:

> The following annotations are added to workload resources, such as Deployments to navigate to the Git repository of the associated application using the edit code decorator:
>  ```yaml
> ...
>   labels:
>     app.openshift.io/vcs-uri: <GIT_REPO_URL>
> ```
> You can also add the following annotation to navigate to a specific branch:
> ```yaml
> ...
>  labels:
>    app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
> ```
However, these annotations should instead be placed under `annotations` instead of `labels` as the url would be an invalid value for a label:
image

We should also provide a list of currently supported runtime types (`<RUNTIME_NAME>`) with icons for the topology node runtime icons configured with:
```yaml
  ...
  labels:
    app.openshift.io/runtime: <RUNTIME_NAME>
```
OR
 ```yaml
 ...
  labels:
    app.kubernetes.io/name: <RUNTIME_NAME>
```
Currently supports the following types:
- django
- dotnet
- drupal
- go-gopher
- golang
- grails
- jboss
- jruby
- js
- nginx
- nodejs
- openjdk
- perl
- phalcon
- php
- python
- quarkus
- rails
- redis
- rh-spring-boot
- rust
- java
- rh-openjdk
- ruby
- spring
- spring-boot

### PR DESCRIPTION
**Pull request by Zaperex**
Fixes #677

### CODE REVIEW

**Comment by Zaperex** [comment]
Should we include the list in the README at all? If so, should we have a separate file for the list when we add more icons?

The main reasoning for this list is just because I wasn't sure why certain types were not loading icons, and didn't know what runtime types would load one.

**Comment by divyanshiGupta** [comment]
cc: @hmanwani-rh

File: plugins/topology/README.md [hunk 1]
```diff
@@ -134,35 +134,62 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
...
 
   ```yaml title="deployment.yaml"
-    ...
-    labels:
-      app.kubernetes.io/name: <RUNTIME_NAME>
+  labels:
+    app.kubernetes.io/name: <RUNTIME_NAME>
   ```
 
+  Currently supported values of `<RUNTIME_NAME>`:
```
Comment (hmanwani-rh) [inline plugins/topology/README.md:162]:
```suggestion
  Supported values of `<RUNTIME_NAME>` include:
```

File: plugins/topology/README.md [hunk 2]
```diff
@@ -134,35 +134,62 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
...
+  - redis
+  - rh-spring-boot
+  - rust
+  - java
+  - rh-openjdk
+  - ruby
+  - spring
+  - spring-boot
+
+  Any other values would result in no icons being rendered for the node.
```
Comment (hmanwani-rh) [inline plugins/topology/README.md:191]:
```suggestion
Other values result in icons not being rendered for the node. 
 
```

**Review by hmanwani-rh — CHANGES_REQUESTED** [review]
@Zaperex Added a couple of suggestions, the rest looks good.

**Comment by divyanshiGupta** [comment]
@Zaperex the PR is not adding any new packages or updating any versions so lets not update `yarn.lock`.

**Comment by Zaperex** [comment]
ah I didn't realize it updated the yarn.lock, I'll fix that

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**